Chrono/Chrono/Common Elements/LottieView.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  LottieView.swift
+//  Chrono
+//
+//  Created by Sagar on 2021-06-22.
+//
+
+import Lottie
+import SwiftUI
+
+struct LottieView: UIViewRepresentable {
+    typealias UIViewType = UIView
+    var filename: String
+
+    func makeUIView(context _: UIViewRepresentableContext<LottieView>) -> UIView {
+        let view = UIView(frame: .zero)
+
+        let animationView = AnimationView()
+        let animation = Animation.named(filename)
```

Comment:
I think we should add a default animation if the provided filename is incorrect, or add a null check to ensure that if the file isn't found, we don't crash.

Reply:
I'm actually thinking of revisiting this wrapper. Maybe look for a better library or something! So I'll note this stuff down and we can try to implement it into the better Lottie impl.